src/algebraic_topology/topological_simplex.lean

```diff
@@ -0,0 +1,84 @@
+/-
+Copyright (c) 2021 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Adam Topaz
+-/
+import algebraic_topology.simplex_category
+import topology.category.Top
+import topology.instances.nnreal
+
+/-!
+# Topological simplices
+
+We define the natural functor from `simplex_category` to `Top` sending `[n]` to the
+topological `n`-simplex.
+This is used to define `Top.to_sSet` in `algebraic_topology.simpliciaL_set`.
+-/
+
+noncomputable theory
+
+namespace simplex_category
+
+open_locale simplicial nnreal big_operators classical
+
+local attribute [instance]
+  category_theory.concrete_category.has_coe_to_sort
+  category_theory.concrete_category.has_coe_to_fun
+
+/-- The topological simplex associated to `x : simplex_category`.
+  This is the object part of the functor `simplex_category.to_Top`. -/
+def to_top (x : simplex_category) := { f : x → ℝ≥0 | ∑ i, f i = 1 }
+
+/-- A morphism in `simplex_category` induces a map on the associated topological spaces. -/
+def map_to_top {x y : simplex_category} (f : x ⟶ y) : x.to_top → y.to_top :=
```

Comment:
I think it makes sense to add a `simp`-lemma for this def.

Reply:
done